utils/merge_and_push_translations.sh

```diff
@@ -97,15 +118,12 @@ python3 utils/fix_formatting.py --lua --dir data/i18n
 python3 utils/fix_formatting.py --lua --dir data/txts
 
 # Undo one-liner diffs of pure timestamps with no other content
-set +x
-for entry in $(git diff --numstat po/ | sed -En 's/^1\t1\t//p'); do
-  if [ -z "$(git diff "$entry" | grep '^[+-][^+-]' | grep -v '^[+-]"POT-Creation-Date:')" ]
-  then
-    echo "Skipping changes to $entry"
-    git checkout $entry
-  fi
-done
-set -x
+undo_oneliner_diffs
+
+if [ -z "$(git status -s)" ]; then
+  echo "Run completed, nothing to commit."
+  exit 0
+fi
```

Comment:
This block shouldn't be moved. The next steps may also introduce changes from translations not in .po format.

Reply:
Oops, sorry. You're right.